sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/FormTrainingClient.java

```diff
@@ -0,0 +1,190 @@
+     */
+    public FormRecognizerServiceVersion getServiceVersion() {
+        return client.getServiceVersion();
+    }
+
+    /**
+     * Create and train a custom model.
+     * <p>Models are trained using documents that are of the following content
+     * type - 'application/pdf', 'image/jpeg', 'image/png', 'image/tiff'.
+     * Other type of content is ignored.
+     * </p>
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support.</p>
+     *
+     * @param fileSourceUrl source URL parameter that is either an externally accessible
+     * Azure storage blob container Uri (preferably a Shared Access Signature Uri).
+     * @param useLabelFile Boolean to specify the use of labeled files for training the model.
+     *
+     * @return A {@link SyncPoller} that polls the training model operation until it has completed, has failed, or has
+     * been cancelled.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public SyncPoller<OperationResult, CustomFormModel> beginTraining(String fileSourceUrl, boolean useLabelFile) {
+        return beginTraining(fileSourceUrl, useLabelFile, false, null, null);
+    }
+
+    /**
+     * Create and train a custom model.
+     * Models are trained using documents that are of the following content type - 'application/pdf',
+     * 'image/jpeg', 'image/png', 'image/tiff'.
+     * Other type of content is ignored.
+     * <p>The service does not support cancellation of the long running operation and returns with an
+     * error message indicating absence of cancellation support.</p>
+     *
+     * @param fileSourceUrl source URL parameter that is either an externally accessible Azure storage
+     * blob container Uri (preferably a Shared Access Signature Uri).
+     * @param useLabelFile Boolean to specify the use of labeled files for training the model.
+     * @param includeSubFolders to indicate if sub folders within the set of prefix folders will
+     * also need to be included when searching for content to be preprocessed.
+     * @param filePrefix A case-sensitive prefix string to filter documents in the source path
+     * for training. For example, when using a Azure storage blob Uri, use the prefix to restrict
+     * sub folders for training.
+     * @param pollInterval Duration between each poll for the operation status. If none is specified, a default of
+     * 5 seconds is used.
+     *
+     * @return A {@link SyncPoller} that polls the extract receipt operation until it
+     * has completed, has failed, or has been cancelled.
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public SyncPoller<OperationResult, CustomFormModel> beginTraining(String fileSourceUrl, boolean useLabelFile,
+        boolean includeSubFolders, String filePrefix, Duration pollInterval) {
+        return client.beginTraining(fileSourceUrl, useLabelFile, includeSubFolders,
```

Comment:
Not sure if this is somewhere else in the code, but how are we handling that by default we want `Prefix=""` and `includeSubFolders=false`?

Reply:
[This overload](https://github.com/Azure/azure-sdk-for-java/pull/9988/files/045af61fb937f554ea05fcfad70561bc72944faa#diff-5ba51b3b13a12cf04e1ad3a9df7decd5R70) sets the includeSubFolders=false for users. And for Prefix we accept it as String so it can be passed as null, don't set it to empty if not specified.

Reply:
Great! thanks for the pointer